src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -273,23 +273,21 @@ public StateFlow build(FlowState.Main initialState) {
             // 2. Def1 unrepresented, Def2 registered
             // 3. Def1 registered, Def 2 unrepresented
             .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT)
-                .onlyIf((respondent1NotRepresented.and(respondent2NotRepresented))
-                            .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
-                            .or(respondent1OrgNotRegistered.negate().and(respondent2NotRepresented))
-                            .and(not(specClaim)))
-            .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_ONE_V_ONE_SPEC)
-                .onlyIf(oneVsOneCase.and(respondent1NotRepresented).and(specClaim))
+            .onlyIf((respondent1NotRepresented.and(respondent2NotRepresented))
+                        .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
+                        .or(respondent1OrgNotRegistered.negate().and(respondent2NotRepresented))
+                        .and(not(specClaim))
+                        .or(multipartyCase.and(respondent1NotRepresented.and(respondent2NotRepresented)
+                                                   .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
+                                                   .or(respondent1OrgNotRegistered.negate().and(respondent2NotRepresented)))
+                                .and(specClaim)))
```

Comment:
Braces should be the same as they were before i.e.  "280-283 are "one block"" was the case before, the block is now just under the same transition rather than a duplicate.

No additional tests as no additional functionality, this transition existed before and is/was tested before